### DISCUSSION TITLE
subruledef issues

### DISCUSSION THREAD

**Discussion by MSlockbo** [hlorenzi/customasm#90] [Q&A] [answered]
Not sure if this is a bug or not, I followed the wiki, but I clearly messed something up:
![image](https://user-images.githubusercontent.com/86361573/123150517-74eadd80-d430-11eb-96cd-cb84e81f660c.png)

Edit:
It works if I put a character in front of the open bracket.

Also, apologies for putting this in the wrong spot, I forgot GitHub has an issues tab now and couldn't figure out how to migrate this post.

**Comment by hlorenzi** [comment, marked as answer]
That's ok, I've moved it to issue #91 !
